src/odemis/util/__init__.py

```diff
@@ -147,27 +147,37 @@ def rect_intersect(ra, rb):
 
 
 def perpendicular_distance(start, end, point):
-    """ Return the perpendicular distance between the line described by start and end, and point """
-
+    """
+    Computes the perpendicular distance between a segment and a point (in 2D space).
+    start (float, float): beginning of the segment
+    end (float, float): end of the segment
+    point (float, float): point anywhere in space
+    return (0 <= float): distance
+    """
     x1, y1 = start
     x2, y2 = end
     x3, y3 = point
 
```

Comment:
I would suggest to rename it to x_start etc. as it is easier to read the code in between.

Reply:
Actually, I prefer not to change the names, as it'd hide the changes required to fix the actual bug.
Ideally, the function readability could be improved, indeed, but in a separate commit (which I'm too lazy to do ;-) ). Maybe writing just `start[0]` and `start[1]` would be as readable.

Reply:
:D